tests/Xamarin.Android.Timing/TimingLogger.cs

```diff
@@ -0,0 +1,358 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.IO;
+using System.Text;
+using System.Xml;
+
+using Microsoft.Build;
+using Microsoft.Build.Evaluation;
+using Microsoft.Build.Framework;
+using Microsoft.Build.Utilities;
+
+namespace Xamarin.Android.Timing
+{
+	public class TimingLogger : Logger
```

Comment:
Should this just go into `build-tools/Xamarin.Android.Tools.BootstrapTasks`? That project already has `Microsoft.Build.Framework`/etc. dependencies.